From e2c7440d31818489c340e018cc24508c51c648e5 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Tue, 13 Oct 2015 15:49:23 +0800
Subject: [PATCH 0561/5242] MLK-11701-1 ARM: dts: imx6sx-sdb: add dts support
 for pxp

commit  197972675826ec64aa96b29872c891664008fb2f from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pxp module on the imx6sx sdb
board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dtsi |    4 ++++
 arch/arm/boot/dts/imx6sx.dtsi     |    1 +
 2 files changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dtsi b/arch/arm/boot/dts/imx6sx-sdb.dtsi
index a0b4692..3499cca 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dtsi
+++ b/arch/arm/boot/dts/imx6sx-sdb.dtsi
@@ -552,6 +552,10 @@
 	status = "okay";
 };
 
+&pxp {
+        status = "okay";
+};
+
 &snvs_poweroff {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 6ea9fc5..af05626 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1311,6 +1311,7 @@
 				};
 
 				pxp: pxp@2218000 {
+					compatible = "fsl,imx6sx-pxp-dma", "fsl,imx6sl-pxp-dma", "fsl,imx6dl-pxp-dma";
 					reg = <0x02218000 0x4000>;
 					interrupts = <GIC_SPI 8 IRQ_TYPE_LEVEL_HIGH>;
 					clocks = <&clks IMX6SX_CLK_PXP_AXI>,
-- 
1.7.9.5

